### PR TITLE
Chat Page, without function

### PR DESCRIPTION
**Pull request by MacksimBondarew**

### CODE REVIEW

**Review by approximator — CHANGES_REQUESTED** [review]
Build doesn't work:

```
/workspace/web (chatPage) $ npm run build

> web@0.0.0 build
> tsc && vite build

src/App.tsx:5:1 - error TS6133: 'Header' is declared but its value is never read.

5 import Header from "./shared/Header/Header";
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Found 1 error in src/App.tsx:5
```

File: src/pages/Chats/ChatGroup/ChatGroup.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import styles from './ChatGroup.module.css'
+import Infrotmation from './Infrotmation/Infrotmation'
```
Comment (approximator) [inline src/pages/Chats/ChatGroup/ChatGroup.tsx:3]: Probably a typo here: `Infrotmation`

Comment (MacksimBondarew) [reply src/pages/Chats/ChatGroup/ChatGroup.tsx:3]: I added home in the App

**Review by waridesu — APPROVED** [review]

File: src/pages/Chats/ChatGroup/ChatGroup.tsx (hunk 1 above)
Comment (approximator) [reply src/pages/Chats/ChatGroup/ChatGroup.tsx:3]: ok, but what does `Infrotmation` mean?

**Comment by MacksimBondarew** [comment]
с

**Pull request merged**